### PR TITLE
Write `cypress` test for `code tab`

### PR DESCRIPTION
**Pull request by Aritra8438**
# Description

This PR adds `cypress` test for `code tab`.

Fixes: #7594

# Checklist:

- [x] The code follows the [coding guidelines](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) of omegaUp.
- [x] The tests were executed and all of them passed.
- [x] If you are creating a feature, the new tests were added.
- [x] If the change is large (> 200 lines), this PR was split into various Pull Requests. It's preferred to create one PR for changes in controllers + unit tests in PHPUnit,  and then another Pull Request for UI + tests in Jest, Cypress or both.

### CODE REVIEW

File: cypress/e2e/problem_creator.cy.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    cy.clearLocalStorage();
+    cy.visit('/');
+  });
+
+  it('Should write and verify the problem statement', () => {
+    const loginOptions: LoginOptions = {
+      username: 'user',
+      password: 'user',
+    };
+    cy.login(loginOptions);
```
Comment (carlosabcs) [inline cypress/e2e/problem_creator.cy.ts:15]: Pass the object directly.

File: cypress/e2e/problem_creator.cy.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      '<h1>Previsualización</h1>\n\n<p>Hello omegaUp!</p>',
+    );
+  });
+
+  it('Should write and verify the problem solution', () => {
+    const loginOptions: LoginOptions = {
+      username: 'user',
+      password: 'user',
+    };
+    cy.login(loginOptions);
```
Comment (carlosabcs) [inline cypress/e2e/problem_creator.cy.ts:35]: Same here.

File: cypress/e2e/problem_creator.cy.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      '<h1>Previsualización</h1>\n\n<p>Hello <strong>solution</strong>!</p>',
+    );
+  });
+
+  it('Should upload code and verify the problem code', () => {
+    const loginOptions: LoginOptions = {
+      username: 'user',
+      password: 'user',
+    };
+    cy.login(loginOptions);
```
Comment (carlosabcs) [inline cypress/e2e/problem_creator.cy.ts:57]: Here.

File: stuff/database_utils.py
Comment (carlosabcs) [inline stuff/database_utils.py:1]: @pabo99 can you please confirm that these changes are right?

**Review by carlosabcs — COMMENTED** [review]
I'm in doubt about the changes to the `docker-compose` command. But maybe @pabo99 can help us checking that.

File: docker-compose.yml [hunk 4]
```diff
@@ -1,3 +1,5 @@
+version: '3.3'
+
```
Comment (pabo99) [inline docker-compose.yml:2]: These lines can be removed since in `docker-compose` v2 this argument is optional

File: stuff/database_utils.py
Comment (pabo99) [reply stuff/database_utils.py:1]: All these changes were applied to update and unify the docker-compose version used throughout the codebase. However, for some reason, Aritra encountered errors when he tried to push his changes.

@Aritra8438, could you please update your branch and then revert all these changes, hoping that the errors have been resolved?

Comment (Aritra8438) [reply stuff/database_utils.py:1]: Sorry, @carlosabcs @pabo, when I push changes, I basically do:
```shell
git revert 210b2f738eeef6ca2c896bcdc96342808d0561b2 --no-commit
```
This time it got staged and pushed.

Sorry for this.

File: docker-compose.yml (hunk 4 above)
Comment (Aritra8438) [reply docker-compose.yml]: It's reverted.

**Review by carlosabcs — APPROVED** [review]
LGTM

**Comment by Aritra8438** [comment]
@heduenas PTAL!

**Review by heduenas — APPROVED** [review]

**Pull request merged**